content/en/integrations/guide/jmxfetch-fips.md

```diff
@@ -0,0 +1,151 @@
+keytool -keystore jmxfetch-keystore -genkey -alias jmxfetch -dname CN=jmxfetch -validity 365 -keyalg ec -storepass changeit
+```
+
+Then export public parts of the generated certificates to separate files:
+
+```
+keytool -keystore java-app-keystore -export -alias java-app -rfc -file java-app-cert.pem -storepass changeit
+keytool -keystore jmxfetch-keystore -export -alias jmxfetch -rfc -file jmxfetch-cert.pem -storepass changeit
+```
+
+Finally, make the certificates trusted by importing them into corresponding trust stores:
+
+```
+keytool -keystore java-app-truststore -import -alias jmxfetch -file jmxfetch-cert.pem -storepass changeit -noprompt
+keytool -keystore jmxfetch-truststore -import -alias java-app -file java-app-cert.pem -storepass changeit -noprompt
+```
+
+{{% /tab %}}
+
+{{% tab "Containers" %}}
+
+Datadog Agent FIPS JMX docker images use BCFKS key store format. PKCS#12 and JKS key store formats used by
+other Java implementations are not supported. `keytool` utility provided in the docker image must be used to
+configure key and trust stores for JMXFetch. `keytool` utility from the Java implementation used to run the
+monitored Java application must be used to configure key and trust stores used by the application.
+
+Pick a location to place the key and trust stores, make it the current directory.
+
+The following commands create two new certificates in two new key stores:
+
+```
+keytool -keystore java-app-keystore -genkey -alias java-app -dname CN=java-app -validity 365 -keyalg ec -storepass changeit
+
+docker run --rm -v $(pwd):/ssl datadog/agent:latest-fips-jmx \
+  keytool -keystore /ssl/jmxfetch-keystore -genkey -alias jmxfetch -dname CN=jmxfetch -validity 365 -keyalg ec -storepass changeit -keypass changeit
+```
+
+Then export public parts of the generated certificates to separate files:
+
+```
+keytool -keystore java-app-keystore -export -alias java-app -rfc -file java-app-cert.pem -storepass changeit
+
+docker run --rm -v $(pwd):/ssl datadog/agent:latest-fips-jmx \
+  keytool -keystore /ssl/jmxfetch-keystore -export -alias jmxfetch -rfc -file /ssl/jmxfetch-cert.pem -storepass changeit
+```
+
+Finally, make the certificates trusted by importing them into corresponding trust stores:
+
+```
+keytool -keystore java-app-truststore -import -alias jmxfetch -file jmxfetch-cert.pem -storepass changeit -noprompt
+
+docker run --rm -v $(pwd):/ssl datadog/agent:latest-fips-jmx \
+  keytool -keystore /ssl/jmxfetch-truststore -import -alias java-app -file /ssl/java-app-cert.pem -storepass changeit -noprompt
+```
+
+{{% /tab %}}
+
+{{< /tabs >}}
+
+### Configuring the Java application
+
+See application documentation for specific instructions about how to configure JMX properties. At least the
+following properties must to be set. Adjust paths as needed to point to actual location of the key and trust
+store files.
```

Comment:
```suggestion
See application documentation for specific instructions about how to configure JMX properties. Add these JMX properties to your application configuration. Adjust the paths to your key and trust store files.
```
Can we add a link to the application documentation we're referring to?

Reply:
There isn't one document to link to, JMXFetch can be used to pull metrics from any Java application. For some off-the-shelf products we have official integrations and those come with more specific instructions, for example [ActiveMQ](https://github.com/DataDog/integrations-core/tree/master/activemq) and [Kafka](https://github.com/DataDog/integrations-core/tree/master/kafka).

Reply:
Noted, appreciate the context.